### ISSUE TITLE
Can not reproduce 35 SAD by your pretrained model

### ISSUE THREAD

**Issue by kfeng123** [Yaoyi-Li/GCA-Matting#10]
Hi Yaoyi,

Thank you very much for this extremely powerful work!

I would like to reproduce your result using the provided pretrained model. I use your code Composition_code.py to generate the test set.  After running your test.sh, however, I only get SAD 39.

The only clue is that it has warnings by libpng like:

> libpng warning: iCCP: known incorrect sRGB profile

But I think this warning should not influence the matting result.

I note that you have emphassis that matlab evaluate code is different from your python. So what's the SAD should it be for python evaluation?

**Comment by Yaoyi-Li** [comment]
Hi,
Could you have a look at this issue https://github.com/Yaoyi-Li/GCA-Matting/issues/4#issuecomment-610884896 ? Can you reproduce their results SAD: 37.8 by the composition code at https://github.com/huochaitiantang/pytorch-deep-image-matting/blob/master/tools/composite.py ? 
If you can reproduce their results, I think there may be some bugs in our composition code that I didn't realize in testing. If you cannot reproduce their results, there may be something wrong with your testing data.
There is no much difference between python and MATLAB SAD evaluation. The main difference is in Conn evaluation code.

**Comment by kfeng123** [comment]
> Hi,
> Could you have a look at this issue [#4 (comment)](https://github.com/Yaoyi-Li/GCA-Matting/issues/4#issuecomment-610884896) ? Can you reproduce their results SAD: 37.8 by the composition code at https://github.com/huochaitiantang/pytorch-deep-image-matting/blob/master/tools/composite.py ?
> If you can reproduce their results, I think there may be some bugs in our composition code that I didn't realize in testing. If you cannot reproduce their results, there may be something wrong with your testing data.
> There is no much difference between python and MATLAB SAD evaluation. The main difference is in Conn evaluation code.

Sorry, it turns out your code can exactly produce your SAD result. It was something wrong with my data. Thank you!
